src/api/app/controllers/concerns/triggerable.rb

```diff
@@ -39,8 +39,14 @@ def set_multibuild_flavor
     @multibuild_container = @package_name.gsub(/.*:/, '') if @package_name.present? && @package_name.include?(':')
   end
 
+  private
+
   def package_from_project_link?
     # a remote package is always included via project link
     !(@package.is_a?(Package) && @package.project == @project)
   end
+
+  def project_links_to_remote
```

Comment:
```suggestion
  def project_links_to_remote?
```
I find the name a little confusing... Without the question mark, I would expect a list of links to remote (which makes no sense). Usually, every method returning a boolean is ending with a question mark.